app/components/article-wrapper.js

```diff
@@ -86,6 +86,10 @@ export default Component.extend(
 			}
 		}),
 
+		featuredVideoVisible: computed('model.hasPortableInfobox', 'model.featuredVideo', function () {
```

Comment:
You can cast it to bool: https://www.emberjs.com/api/classes/Ember.computed.html#method_bool